frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -222,35 +222,35 @@ const fetchNodeMetrics = (): Promise<NodeMetrics> => {
   const metrics = [
     {
       key: 'usedMemory',
-      query: 'sum by (instance) (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes)',
```

Comment:
I removed all of these sum by's because all of these queries should only return one timeseries now that the storage one is fixed. If more than one was being returned, a sum was likely not the right aggregation method anyway.